account_account_constraint_code/tests/test_account_account.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2022 Acsone SA (http://www.acsone.eu/)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from datetime import datetime
+
+from odoo.exceptions import ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class TestAccountAccount(TransactionCase):
+    def setUp(self):
+        super().setUp()
+        self.misc_journal = self.env["account.journal"].create(
+            {
+                "name": "Test Journal Move name seq",
+                "code": "ADLM",
+                "type": "general",
+                "company_id": self.env.company.id,
+            }
+        )
+        self.purchase_journal = self.env["account.journal"].create(
+            {
+                "name": "Test Purchase Journal Move name seq",
+                "code": "ADLP",
+                "type": "purchase",
+                "company_id": self.env.company.id,
+                "refund_sequence": True,
+            }
+        )
+        self.accounts = self.env["account.account"].search(
+            [("company_id", "=", self.env.company.id)], limit=2
+        )
+        self.account1 = self.accounts[0]
+        self.account2 = self.accounts[1]
+        self.date = datetime.now()
```

Comment:
```
from odoo import fields
...
self.date = fields.Datetime.now()
```
Minor change but better to use what Odoo use instead of importing lib (event if builtin)